pkg/querier/queryrange/queryrange.proto

```diff
@@ -160,14 +161,27 @@ message QuantileSketchResponse {
 }
 
 message QueryResponse {
```

Comment:
After some issues I think all requests and responses should have a `SetMetadata` and `GetMetadata` method.